scripts/packaging/createupmpackages.ps1

```diff
@@ -131,6 +119,13 @@ foreach ($entry in $packages.GetEnumerator()) {
         # the folder.
         Start-Process -FilePath "$PSHOME\powershell.exe" -ArgumentList "$scriptPath\examplesfolderpreupm.ps1 -PackageRoot $packagePath" -NoNewWindow -Wait
     }
+    elseif ($packageName -eq "extensions") {
+        # The extensions folder contains one or more folders that provide their own examples. In order
+        # to perform the necessary preparation, without overly complicating this script, we will use a
+        # helper script to prepare the folder.
+        Start-Process -FilePath "$PSHOME\powershell.exe" -ArgumentList "$scriptPath\extensionsfolderpreupm.ps1 -PackageRoot $packagePath" -NoNewWindow -Wait
+    }
+    # todo: similar script for extensions

```

Comment:
I'm a bit confused by this todo and the fact that we already have a packageName that is literally called extensions - it seems like this would be a stale todo.

If the intent is to say "hey, here's is where you could add other scripts/handlers, then you could say add another elseif statement modelled after above.

But actually in that case I assume that we'd get rid of some copy pasta because the only difference here are the script names and the package name (which we could do using a hashtable)

Reply:
it's a leftover note to remind me to add the section

Reply:
What's the section that you're going to add here?

There is already a else if statement that literally says "extensions" and this comment says "extensions".

Reminders to self easily get lost (last example being the TODO that went in the release notes that went out).

Reply:
> What's the section that you're going to add here?

The one above.

this is now fixed